### ISSUE TITLE
A11y_Model Builder_advanced data option_validation data_Name role value: Name is not provided for "Browse a file" Button.

### ISSUE THREAD

**Issue by Harika1702** [dotnet/machinelearning-modelbuilder#2658]
### GitHub Tags
#WCAG4.1.2; #A11ySev3; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #FTP; #Win32; #Benchmark; #Name Role Value;

### Environment Details
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise Build 22624.1755

### Repro Steps
1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Complete Scenario and Environment setup steps.
7. Add Data screen will appear. Tab till Advance data option.
8. Advance data option screen will appear, tab till Validation data and press enter.
9. Tab till "Browse a file" Control.
10. Now verify that Name is not provided for "Browse a file" Button.

### Actual Result
Name is not provided for "Browse a file" Button.
**Note: we are facing similar issue throughout the application.**

### Expected Result
Name should be provided for "Browse a file" Button.

### User Impact
Screen reader users will not be able to understand the name of the control if it is not defined.

### Attachment
<img width="960" alt="Bug_Name" src="https://github.com/dotnet/machinelearning-modelbuilder/assets/100270074/090bc84d-25b8-4e5f-a7c4-7ae044aecdc2">

**Comment by zewditu** [comment]
@Harika1702, ready to validate by Model builder version 16.17.0.2332602.

**Comment by SachinChavhan317** [comment]
Verified the bug on below model builder version.
Version:17.17.0.2332602

The issue is fixed, hence closing the bug. #closed;

Adding fixed attachment for reference.
![2658_Fixed](https://github.com/dotnet/machinelearning-modelbuilder/assets/86241494/3ea59ec1-754f-48ed-a5a2-81ee9f759d54)
